### ISSUE TITLE
Comments on openjournals/joss-reviews#6366 article proof text

### ISSUE THREAD

**Issue by ifoxfoot** [equitable-equations/fqar#6]
openjournals/joss-reviews#6366 

Line 6-7:  Floristic Quality Assessment (FQA) is a standardized method for rating the ecological value of natural areas based on the plant species found within them (Spyreas, 2015). 
- I would use a different citation. The citation you provided links to a conference, but not specifically to any FQA information. I would perhaps use [(Spyreas, 2019)](https://doi.org/10.1002/ecs2.2825) and one of the Swink and Wilhelm papers (1979, 1994).

Line 7-8: Each species considered native to a particular region is assigned a coefficient of conservatism, C, on a scale of 0-10 by experts in local flora.
- Not necessarily. Many regional FQA lists contain non-native species as well. Typically non-native species are always assigned a C score of 0. I would reword this sentence to be more factually correct.

Line 12-15: This native mean C-value, sometimes weighted by the total number of plant species identified to give the so-called floristic quality index (Bowles & Jones, 2006), is frequently used by land managers and other agents to quantify an area’s state of conservancy (Zinnen, 2021).
- Again, many people/FQA lists include non-native species. Also, why not cite Swink and Wilhelm 1994 for the floristic quality index metric. I believe they invented the metric. The citation would be "Swink, F., & Wilhelm, G. (1994). Plants of the Chicago region. Indiana Academy of Science."

Line 30-32: While it facilitates storage and sharing, universalfqa.org does not provide users any tools for make constructive use of its data beyond simple viewing and downloading of individual assessments. 
- I feel this is a bit misleading. In line 16-17 you state that users "upload their floristic quality assessments" implying that an assessment is just a list of species (do you mean they upload inventories?). Then you state that the universal calculator only is used for "simple viewing and downloading of individual assessments," implying that the universal fqa calculator is only good for storing and viewing lists of species. In reality, the universal FQA calculator calculates a wide variety of metrics for the user. I would clarify what an assessment is and clarify the purpose/utility of the universal calculator. 

line 40-41: which species in a given database might be misclassified? Based on their co-occurrence profiles, might some be more or less conservative than previously thought?
- I'm not sure you can determine if a species is misclassified based on this type of analysis. It is interesting...but there is no spatial information in most FQA assessments. So two species might be in the same assessment but they might not be in the same micro habitat. Also I'm not sure that a high tolerance for disturbance precludes a plant from growing in an undisturbed area...I do think this is an interesting analysis but there are a lot of limitations. Food for thought.

**Comment by equitable-equations** [comment]
@ifoxfoot thank you for these thoughtful comments! I've taken them into account and posted a revision [here](https://github.com/equitable-equations/fqar/tree/joss_revision/paper). Some specific responses to your points:

### Lines 6-7: I would use a different citation.

Response: The recommended citation has been added.

### Lines 7-8: I would reword this sentence to be more factually correct.

Response: A parenthetical has been added to explicitly mention that non-native species are assigned a C-value of 0. 

### Lines 12-15: Also, why not cite Swink and Wilhelm 1994 for the floristic quality index metric. 

Response: The Wilhelm citation has been added earlier, as suggested. At this point in the paper, we feel that the Zinnen reference is appropriate since it explicitly describes how FQA is actually used in the field today. 

### Lines 30-32: Clarify what an assessment is and clarify the purpose/utility of the universal calculator.

Response: This paragraph has been recast to better reflect the utility of the universalFQA calculator.

### Lines 40-41: I'm not sure you can determine if a species is misclassified based on this type of analysis.

Response: In collaboration with local botanists, the authors are in the process of considering exactly this question (and others like it) for a forthcoming paper. Our general premise is that as long as two species are found simultaneously by a practitioner who chooses to include them both in a single site assessment, they may reasonably be taken to be co-occurring. We recognize the potential limitations of such studies but are also finding them to be both useful and insightful.

A spacial analysis would be incredibly valuable and is an important direction for future study. Please let me know if you'd like to collaborate on this!

**Comment by equitable-equations** [comment]
@ifoxfoot yes to all. Specifically:

- I've added the word "generally" to allow for the unusual handling of non-native species

- The paragraph you reference has been recast, ending "...though it has so far gone largely unexplored due to a lack of both technical tools for interacting programmatically with the repository and accessible workflows for analyzing the floristic quality data housed there."

- I'm hoping to tackle iNaturalist next!

All changes will be pushed to the repo shortly.
